sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/transfer/UploadRequest.java

```diff
@@ -63,14 +67,23 @@ String getContainerName() {
     }
 
     /**
-     * Get the name of the target blob holding uploaded file.
+     * Get whether or not the library should calculate the md5 and send it for the service to verify.
      *
-     * @return The blob name.
+     * @return Whether or not the library should calculate the md5 and send it for the service to verify.
      */
     String getBlobName() {
         return this.blobName;
     }
 
+    /**
+     * Get the name of the target blob holding uploaded file.
+     *
+     * @return The blob name.
+     */
```

Comment:
These two Javadoc entries should be swapped.

Reply:
Whoops thanks for catching that